### PR TITLE
Ot217 85

### PR DESCRIPTION
**Pull request by filburt88**
*Created global variable on environment file and implemented on news service.

*PLEASE DON'T MERGE AS SLIDES COMPONENTS WILL NOT BE OPERATIVE (waiting for Francisco to finish slides global variables)
code-snapshot2
code-snapshot

### CODE REVIEW

File: src/app/core/services/news.service.ts [hunk 1]
```diff
@@ -2,34 +2,38 @@ import { HttpClient } from '@angular/common/http';
...
 
+  endpoint:string = environment.news;
+
   public constructor(http:HttpClient)  {
     super(http);
   }
 
   postNews<News>(news:any):Observable<any>{
-    return this.post<News>(news);
+    console.log(this.endpoint)
```
Comment (anagon88) [inline src/app/core/services/news.service.ts:19]: borrar linea

File: src/environments/env.example.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const environmentPlaceHolder = {
```
Comment (anagon88) [inline src/environments/env.example.ts:1]: cambiarle el nombre al archivo a "environment.prod.ts"

File: src/environments/env.example.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    production: false,
```
Comment (anagon88) [inline src/environments/env.example.ts:2]: cambiar a true

**Review by anagon88 — CHANGES_REQUESTED** [review]

File: src/app/core/services/news.service.ts (hunk 1 above)
Comment (filburt88) [reply src/app/core/services/news.service.ts]: Done

**Review by anagon88 — APPROVED** [review]

**Pull request merged**